iceoryx_posh/test/fuzztests/README.md

```diff
@@ -0,0 +1,49 @@
+#Fuzzing Iceoryx
+
+##What is fuzzing?
+Fuzzing is a method for dynamic code analysis which can be used to find several types of bugs which could also be security vulnerabilities. These vulnerabilities which could lead to a crash such as buffer overflow, use-after-free, dangling pointer and more can be found. 
+More information about fuzzing can for example be found [here](https://owasp.org/www-community/Fuzzing "OWASP Fuzzing"). 
+
+##Why should you use fuzzing?
+1. **It is easy to use** - These fuzz wrappers provide you a possibility to integrate a fuzzer very fast. It can then run for several hours or days and find bugs automatically. 
+2. **It is effective** - Lots of security vulnerabilities in the wild are found with fuzzers. 
+3. **It has a low false positive rate** - If the program crashes, then it is very likely that something in your code was wrong. 
+
+
+##How to build
+First of all, you need to install the fuzzer you want to use. The american fuzzy lop (afl) for example can be found [here](https://github.com/google/AFL "Github AFL").  
```

Comment:
Please add a link to the installation manual

Reply:
Done